engine/src/main/java/org/terasology/rendering/dag/nodes/CopyImageToScreenNode.java

```diff
@@ -60,12 +65,15 @@ public void initialise() {
     public void process() {
         PerformanceMonitor.startActivity("rendering/copyImageToScreen");
         sceneFinalFbo.bindTexture(); // TODO: Convert to a StateChange
+        glViewport(0, 0, displayWidth, displayHeight); // TODO: Convert to a StateChange
```

Comment:
Not really. The problem is, SetViewportToSizeOf should only consider the size of FBO, and not whether the size is because the display is actually that size or because the display * scale is that size. Which is the case right now. We can make a new SetViewportToSizeOfDisplay StateChange, but that's it. I tried to integrate this into SVTSO, but found no plausible way.